.gitattributes

```diff
@@ -0,0 +1 @@
+* text=auto eol=lf
```

Comment:
I guess we can keep this, due to the fact that I develop on mac and windows for this app I configured my git configs to always use LF as file endings, but that's a personal setting